odata-csdl/13 Entity Container.md

```diff
@@ -446,6 +446,11 @@ that differ only in a type-cast segment, allowing to bind instances of
 different sub-types to different targets. If paths differ only in
 type-cast segments, the most specific path applies.
 
+Elsewhere in this specification ([#OData-Protocol#CreateanEntity], [#OData-Protocol#UpsertanEntity],
+[#OData-URL#AddressingEntities]) an [instance path](#PathExpressions) that starts
+with an entity container is said to _match_ a navigation property path binding
```

Comment:
Should be starts at, not starts with (the entity container is not part of the url)